### PR TITLE
Changed %lld to use mingw supported PRId64

### PR DESCRIPTION
**Pull request by kpayson64**
This resolves part of the ruby build errors blocking release.

### CODE REVIEW

File: src/core/lib/profiling/basic_timers.c [hunk 1]
```diff
@@ -141,7 +141,7 @@ static void write_log(gpr_timer_log *log) {
       entry->tm = gpr_time_0(entry->tm.clock_type);
     }
     fprintf(output_file,
-            "{\"t\": %lld.%09d, \"thd\": \"%d\", \"type\": \"%c\", \"tag\": "
+            "{\"t\": %"PRId64".%09d, \"thd\": \"%d\", \"type\": \"%c\", \"tag\": "
             "\"%s\", \"file\": \"%s\", \"line\": %d, \"imp\": %d}\n",
             (long long)entry->tm.tv_sec, (int)entry->tm.tv_nsec, entry->thd,
```
Comment (jtattermusch) [inline src/core/lib/profiling/basic_timers.c]: Remove (long long) for tv_sec?

File: src/core/ext/client_config/channel_connectivity.c [hunk 2]
```diff
@@ -189,9 +189,9 @@ void grpc_channel_watch_connectivity_state(
   GRPC_API_TRACE(
       "grpc_channel_watch_connectivity_state("
       "channel=%p, last_observed_state=%d, "
-      "deadline=gpr_timespec { tv_sec: %lld, tv_nsec: %d, clock_type: %d }, "
+      "deadline=gpr_timespec { tv_sec: %"PRId64", tv_nsec: %d, clock_type: %d }, "
       "cq=%p, tag=%p)",
-      7, (channel, (int)last_observed_state, (long long)deadline.tv_sec,
+      7, (channel, (int)last_observed_state, deadline.tv_sec,
           (int)deadline.tv_nsec, (int)deadline.clock_type, cq, tag));
```
Comment (jtattermusch) [inline src/core/ext/client_config/channel_connectivity.c]: for tv_nsec, do you need the cast?

It looks like that in other places it works without (int) cast. 
https://github.com/grpc/grpc/search?utf8=%E2%9C%93&q=PRId64

**Comment by kpayson64** [comment]
Addressed comments.

**Comment by jtattermusch** [comment]
LGTM.

**Pull request merged**